.github/workflows/bld_maven.yml

```diff
@@ -0,0 +1,218 @@
+name: bld_maven
+run-name: bld-{{ inputs.artifact_name }}
+
+permissions:
+  checks: write
+  contents: read
+  issues: read
+  pull-requests: write
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'Name of the artifact to build'
+        required: false
+        default: "orcid-message-listener"
+        type: string
+      version_tag:
+        description: 'Name of the tag to build'
+        required: false
+        default: "v2.0.1"
```

Comment:
@gilesw why do we set a default? Lets set this field as requiered and always ask for a version.